.github/workflows/build-and-test.yml

```
@@ -0,0 +1,21 @@
+name: Build and Test
+on: [pull_request, push]
+jobs:
+  build-and-test:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        include:
+          - DISTRO: "tomcat"
+            EE: "false"
+          - DISTRO: "wildfly"
+            EE: "false"
+          - DISTRO: "run"
+            EE: "false"
```

There's not really a need for a complex matrix if `EE` is always false.
```suggestion
      matrix:
        DISTRO: [tomcat, wildfly, run]
```

---

thanks, good idea!